src/main/resources/assets/react/documentTypes/DocumentTypes.tsx

```diff
@@ -31,15 +40,15 @@ function getDefaultModalState(open = false) {
 export function DocumentTypes({
 	servicesBaseUrl
 }) {
-	const [updatedAt, setUpdatedAt] = React.useState(new moment());
+	const [updatedAt, setUpdatedAt] = React.useState(moment());
 	const [durationSinceLastUpdate, setDurationSinceLastUpdate] = React.useState('');
 
 	const [boolPoll, setBoolPoll] = React.useState(true);
 	const [globalFields, setGlobalFields] = React.useState([]);
 	const [documentTypes, setDocumentTypes] = React.useState([]);
 
 	// The modal state should be handled by newOrEditDocumentTypeModal
-	const [newOrEditModalState, setNewOrEditModalState] = React.useState(getDefaultModalState());
+	const [newOrEditModalState, setNewOrEditModalState] = React.useState(getDefaultModalState() as DocumentTypeModal);
```

Comment:
Typing the return value
function getDefaultModalState(open = false) :DocumentTypeModal {